purchase_security/security/security.xml

```diff
@@ -30,6 +38,15 @@
             >['|',('user_id','=',user.id),('user_id','=',False)]</field>
             <field name="groups" eval="[(4,ref('group_purchase_own_orders'))]" />
         </record>
+        <record model="ir.rule" id="purchase_order_group_purchase_order_group_orders">
+            <field name="name">View purchase orders (group responsible)</field>
+            <field name="model_id" ref="purchase.model_purchase_order" />
+            <field
+                name="domain_force"
+            >['|', ('team_id', '=', user.purchase_team_id.id),('team_id', '=', False)]
```

Comment:
That field `purchase_team_id` on `res.users` doesn't even exists. How is this supposed to work?

Reply:
Thanks for the comments, can you check if now it is correct?
I will improve the test coverage after.